examples/src/main/java/io/dapr/examples/invoke/http/README.md

```diff
@@ -0,0 +1,128 @@
+# Method invocation Sample
+
+In this sample, we'll create a two java applications: An exposer service application which exposes a method and a client application which will invoke the method from demo service using Dapr.
+This sample includes:
+
+* ExposerService (Initializes the ExposerServiceController)
+* ExposerServiceController (Handles the remote invoking call)
+* InvokeClient (Invokes the exposed method from DemoService)
+
+Visit [this](https://github.com/dapr/docs/blob/master/concepts/service-invocation/service-invocation.md) link for more information about Dapr and service invocation.
+ 
+## Remote invocation using the Java-SDK
+
+This sample uses the Client provided in Dapr Java SDK for exposing and invoking a remote method.  
```

Comment:
For exposing the endpoint, it is not using the SDK anymore. We are using Springboot but not the SDK directly.

Reply:
For invoking we still use the SDK.